config/backup.conf

```diff
@@ -43,8 +42,20 @@
 # postgres=postgresql:5432/TheOrgBook_Database
 # mongo=mender-mongodb:27017/useradm
 # postgres=wallet-db/tob_issuer
-# mssql=pims-db-dev:1433/pims
 #
 # 0 1 * * * default ./backup.sh -s
 # 0 4 * * * default ./backup.sh -s -v all
-# ============================================================
\ No newline at end of file
+# ============================================================
+postgres=issuer-kit-wallet/agent_esr1_wallet
+postgres=issuer-kit-wallet/agent_esr2_wallet
+postgres=issuer-kit-wallet/agent_healthbc_wallet
+postgres=issuer-kit-wallet/agent_inoculation_wallet
+postgres=issuer-kit-wallet/agent_medlab_wallet
+postgres=issuer-kit-wallet/agent_openvp_wallet
+
+
+# Schedule backup for 1am PST; system TZ is PST
+# 0 1 * * * default ./backup.sh -s
+
+# Schedule backup verification for 4am PST; system TZ is PST
+# 0 4 * * * default ./backup.sh -s -v all
```

Comment:
This file should be reverted to the original, without changes that are deployment specific.